### PR TITLE
Leverage new `abi-wait-complete`

### PR DESCRIPTION
**Pull request by omertuc**
See https://github.com/rh-ecosystem-edge/bip-orchestrate-vm/pull/11 for
more info. Only merge after the above merges

### CODE REVIEW

**Review by eranco74 — COMMENTED** [review]
.

File: Makefile [hunk 1]
```diff
@@ -345,6 +345,7 @@ wait-for-install-complete:
 	@until [ "$$($(oc) get clusterversion -o jsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}')" == "True" ]; do \
 			echo -n .; sleep 10; \
 	done; \
+	make -C $(SNO_DIR) abi-wait-complete INSTALLER_WORKDIR=workdir-$(CLUSTER)a; \
```
Comment (eranco74) [inline Makefile:348]: typo

Comment (eranco74) [reply Makefile:348]: Also, it should be:
INSTALLER_WORKDIR=workdir-$(VM_NAME)

Comment (omertuc) [reply Makefile]: Nope that wouldn't work, I can explain why if you're interesetd

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Pull request merged**